flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/util/HiveTableUtil.java

```diff
@@ -146,15 +150,31 @@ public static ObjectInspector getObjectInspector(DataType flinkType) throws IOEx
 		return getObjectInspector(HiveTypeUtil.toHiveTypeInfo(flinkType));
 	}
 
-	// TODO: reuse Hive's TypeInfoUtils?
 	private static ObjectInspector getObjectInspector(TypeInfo type) throws IOException {
```

Comment:
`HiveInspectors` also has such a method. Can we move the whole thing to `HiveInspectors` so we have a single place to do all kinds of conversions?